pos_container_deposit/tests/test_pos_container_deposit_frontend.py

```diff
@@ -0,0 +1,14 @@
+from odoo.tests.common import tagged
+
+from odoo.addons.point_of_sale.tests.test_frontend import TestPointOfSaleHttpCommon
+
+
+@tagged("post_install", "-at_install")
+class TestPosContainerDeposit(TestPointOfSaleHttpCommon):
+    def run_tour(self):
+        self.main_pos_config.open_ui()
+        self.start_tour(
```

Comment:
I'm surprised about this codecov message here

> Codecov
/ codecov/patch
>
> pos_container_deposit/tests/test_pos_container_deposit_frontend.py#L9-L10
>
> Added lines #L9 - L10 were not covered by tests

These don't seem to be running in our CI 🤔

Reply:
all of them are [skipped](https://github.com/OCA/pos/actions/runs/12504113912/job/34885411105?pr=1278#step:8:412) because we don't have the expected coa - looks like installing `l10n_fr` is causing this, because this suppresses installing l10n_generic_coa which the base class [checks](https://github.com/OCA/OCB/blob/16.0/addons/account/tests/common.py#L37) for.

Maybe we should make [pos_ticket_extra_company_info_l10n_fr](https://github.com/OCA/pos/tree/16.0/pos_ticket_extra_company_info_l10n_fr) a rebel module for this reason?

Reply:
cc @legalsylvain

Reply:
> Maybe we should make [pos_ticket_extra_company_info_l10n_fr](https://github.com/OCA/pos/tree/16.0/pos_ticket_extra_company_info_l10n_fr) a rebel module for this reason?

no problem with such exclusion. 
Note: It's a temporary problem because l10n_generic_coa disappeared in V18, and it's better as it.